views_api.py

```diff
@@ -275,12 +276,18 @@ async def api_check_fiat_rate(currency):
 
 ## ITEMS
 @tpos_ext.put("/api/v1/tposs/{tpos_id}/items", status_code=HTTPStatus.CREATED)
-async def api_tpos_create_items(data: CreateUpdateItemData, tpos_id: str):
+async def api_tpos_create_items(
+    data: CreateUpdateItemData,
+    tpos_id: str,
+    wallet: WalletTypeInfo = Depends(require_invoice_key),
```

Comment:
shouldn't this be an `require_admin_key` because it is a write operation?

Reply:
ok